src/main/java/com/fasterxml/jackson/databind/jdk14/JDK14Util.java

```diff
@@ -150,25 +148,25 @@ public RawTypeName(Class<?> rt, String n) {
     }
 
     static class CreatorLocator {
-        protected final BeanDescription _beanDesc;
```

Comment:
Same as above, please do not change these. Over time I have learned that information from more expressive types (BeanDescription is superset of AnnotatedClass, DeserializationContext similarly superset of MapperConfig) is safer to pass around -- I consistently needed to upgrade parameters.